### PR TITLE
update package; remove ambiguous arrow rule; expand authors

### PR DESCRIPTION
**Pull request by mskalandunas**
/review @imbrianj @lrnwytt @matthewspeck

### CODE REVIEW

File: es6.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ module.exports = {
         "constructor-super": "error",
         "generator-star-spacing": "error",
         "no-class-assign": "error",
-        "no-confusing-arrow": "error",
```
Comment (imbrianj) [inline es6.js:7]: Why this removal?

**Comment by imbrianj** [comment]
lgtm

File: es6.js (hunk 1 above)
Comment (mskalandunas) [reply es6.js:7]: Because it's a really dumb rule that blocks on completely legit JS.

Comment (mskalandunas) [reply es6.js:7]: From ESLint:

```
// The intent is not clear
var x = a => 1 ? 2 : 3;
```

It's a poorly written function but we should be catching those in code review anyway. It blocks  things like this:

```
const fn = a => a ? 'yeet' : 'peet';
```

Which it definitely should not.

**Pull request merged**